sdk/src/client/node_api/indexer/routes.rs

```diff
@@ -43,7 +43,7 @@ impl ClientInner {
     /// Get alias outputs filtered by the given parameters.
     /// GET with query parameter returns all outputIDs that fit these filter criteria.
     /// Query parameters: "stateController", "governor", "issuer", "sender", "createdBefore", "createdAfter"
-    /// Returns an empty list if no results are found.
+    /// Returns Err(Node(NotFound) if not results are found.
```

Comment:
Same here and below.

Reply:
samee

Reply:
did change not -> no, or was that what you meant?